bridge-ext/src/main/java/com/liferay/faces/bridge/ext/config/internal/LiferayPortletConfigParam.java

```diff
@@ -54,7 +54,23 @@
 	 * rendered for each resource which has a name. The id includes the name and the library (if it exists) of the
 	 * resource in order to uniquely identify the resource. The default value is true.
 	 */
-	RenderHeadResourceIds("com.liferay.faces.bridge.ext.renderHeadResourceIds", true);
+	RenderHeadResourceIds("com.liferay.faces.bridge.ext.renderHeadResourceIds", true),
+
+	/**
+	 * The default value of the data-senna-track attribute for PrimeFaces CSS resources. Defaults to "permanent" in
+	 * order to ensure that PrimeFaces CSS remains on the page after SennaJS navigation (in order to hide PrimeFaces
+	 * elements that may not have been cleaned up). This value can be overridden on a per-component basis by setting the
+	 * data-senna-track attribute as a JSF pass-through attribute in your view or via {@link
+	 * javax.faces.component.UIComponent#getPassThroughAttributes()}. Setting this value to the special case value of
+	 * {@link #NO_DEFAULT_VALUE} causes the data-senna-track attribute not to be rendered (unless data-senna-track was
+	 * set as a pass-through attribute on the component).
+	 */
+	PrimeFacesCSSDefaultDataSennaTrackValue("com.liferay.faces.bridge.ext.primeFacesCSSDefaultDataSennaTrackValue",
+		"permanent");
+
+	// Public Constants
+	public static final String NO_DEFAULT_VALUE =
+		"com.liferay.faces.bridge.ext.primeFacesCSSDefaultDataSennaTrackValue.noDefaultValue";
```

Comment:
@stiemannkj1 Having an init-param name be prefixed with "com.liferay.faces.xyz" is very ordinary, but not for the value. Can `NO_DEFAULT_VALUE` be something simpler like an empty string? What are the possible values other than `permanent`?